<launch>
	<node pkg="ferns_ros" type="detector" name="ferns_detector" output="screen">
	  <param name = "image_topic" type="string" value = "/camera/rgb/image_color" />
    <param name = "show_cv_window" type="bool" value = "true" />
    <param name = "model" type="string" value = "/home/vicos/ROS/ferns_data/book.jpg" />
	</node>
</launch>
